src/components/Account/AccountButton.js

```diff
@@ -12,6 +12,8 @@ import {
 import { useWallet } from 'use-wallet'
 import HeaderModule from '../Header/HeaderModule'
 
+const IPFS_BASE_URL = 'https://ipfs.io/ipfs/'
```

Comment:
Oh interesting, this is for 3box profiles? They don't get propagated into our node fast enough?